aspnetcore/data/entity-framework-6.md

```diff
@@ -16,126 +16,47 @@ uid: data/entity-framework-6
 
 By [Paweł Grudzień](https://github.com/pgrudzien12), [Damien Pontifex](https://github.com/DamienPontifex), and [Tom Dykstra](https://github.com/tdykstra)
 
-This article will show you how to use Entity Framework 6 inside an ASP.NET Core application.
+This article shows how to use Entity Framework 6 in an ASP.NET Core application.
 
 ## Overview
 
 To use Entity Framework 6, your project has to compile against the full .NET Framework, as Entity Framework 6 does not support .NET Core. If you need cross-platform features you will need to upgrade to [Entity Framework Core](https://docs.efproject.net).
 
-The recommended way to use Entity Framework 6 in an ASP.NET Core 1.0 application is to put the EF6 context and model classes in a class library project (*.csproj* project file) that targets the full framework. Add a reference to the class library from the ASP.NET Core project. See the sample [Visual Studio solution with EF6 and ASP.NET Core projects](https://github.com/aspnet/Docs/tree/master/aspnetcore/data/entity-framework-6/sample/).
+The recommended way to use Entity Framework 6 in an ASP.NET Core application is to put the EF6 context and model classes in a class library project that targets the full framework. Add a reference to the class library from the ASP.NET Core project. See the sample [Visual Studio solution with EF6 and ASP.NET Core projects](https://github.com/aspnet/Docs/tree/master/aspnetcore/data/entity-framework-6/sample/).
 
-It's not feasible to put an EF6 context in an ASP.NET Core 1.0 project because *.xproj*-based projects don't support all of the functionality that EF6 commands such as *Enable-Migrations* require. In a future release of ASP.NET Core, Core projects will be based on *.csproj* files, and at that time you'll be able to include an EF6 context directly in an ASP.NET Core project.
+It's not feasible to put an EF6 context in an ASP.NET Core project because .NET Core projects in Visual Studio 2017 don't support all of the functionality that EF6 commands such as *Enable-Migrations* require.
```

Comment:
I'm not a fan of feasible. Above OK but consider:
EF6 cannot be use in a  ASP.NET Core project.  .NET Core projects in Visual Studio 2017 don't support all of the functionality that EF6 commands such as *Enable-Migrations* require.

Also, why is it dependent on VS17?  You can use the CLI to run EF commands. Is it true to just say:

EF6 cannot be use in a  ASP.NET Core project.  .NET Core projects ~~in Visual Studio 2017~~ don't support all of the functionality that EF6 commands such as *Enable-Migrations* require.